pkg/build/api/types.go

```diff
@@ -509,6 +509,13 @@ type SourceBuildStrategy struct {
 
 	// ForcePull describes if the builder should pull the images from registry prior to building.
 	ForcePull bool
+
+	// RuntimeImage is the optinal image that is used to run an application
+	// without unneeded dependencies installed.
+	RuntimeImage *kapi.ObjectReference
+
+	// RuntimeArtifacts holds the mapping between the artifacts paths on the builder image and the paths on the runtime image where they should be copied.
```

Comment:
i'm not particularly satisfied with the docs, it was more to satisfy update-generated-swagger-desc, so i might rephrase this completely.